subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -532,4 +533,28 @@ public void setArchiveBase(PathBase archiveBase) {
     public Property<Integer> getNetworkTimeout() {
         return networkTimeout;
     }
+
+    /**
+     * Sets if the task will validate the distribution url that has been configured.
+     *
+     * @since 8.2
+     */
+    @Incubating
+    @Option(option = "validate-url", description = "Sets task to validate the configured distribution url")
+    public void setValidateDistributionUrl(Boolean validateDistributionUrl) {
+        this.validateDistributionUrl.set(validateDistributionUrl);
+    }
+
+    /**
+     * Indicates if this task will validate the distribution url that has been configured.
+     *
+     * @since 8.2
+     * @return whether this task will validate the distribution url
+     */
+    @Incubating
+    @Input
+    @Optional
+    public Property<Boolean> getValidateDistributionUrl() {
+        return validateDistributionUrl;
+    }
```

Comment:
We can rely on Gradle to generate a managed property for us:
```suggestion
    @Incubating
    @Input
    @Option(option = "validate-url", description = "Sets task to validate the configured distribution url")
    public abstract Property<Boolean> getValidateDistributionUrl();
```

This gets rid of the field and implementation here. 

I don't think this property can be marked `@Optional` because we always expect there to be a value (true or false).